### PR TITLE
Enable creating releases with makefile

### PR DESCRIPTION
**Pull request by marusak**
It has been a while since last release was done. This could help making releases easier.

### CODE REVIEW

**Comment by marusak** [comment]
retest this please

**Comment by jfilak** [comment]
Does it correctly update CHANGELOG.md? I tuned this code for ABRT.

**Comment by marusak** [comment]
You have to specify "correctly":) But it does update CHANGELOG.md since it is almost the same code as is in ABRT. When I run it as `make release-major` it adds 3 lines into CHANGELOG.md:
`-## [Unreleased][unreleased]
+## [Unreleased]
+
+## [1.0.0][unreleased]
`

**Comment by jfilak** [comment]
Well, that does not look good. Please have a look at this:
http://keepachangelog.com/en/0.3.0/

Besides there is a flaw in the code. New release entry misses date. Let me first fix this issue in libreport/abrt and take over the code after that. Or you can fix it on your own, I don't mind - but I admit it is my fault ;)

**Comment by jfilak** [comment]
Date added:
https://github.com/abrt/libreport/commit/44331c82c64758ecad64185709a20fabe7ed673e

**Comment by jfilak** [comment]
Ah, the make target does not work because it expects:
```[Unreleased]```

but ```CHANGELOG.md``` contains:
```[Unreleased][unreleased]```

@sorki Would you mind us updating the format of ```CHANGELOG.md``` to match the format described [here](http://keepachangelog.com/en/0.3.0/)?

**Comment by marusak** [comment]
Ok, I updated it. Now it works better. `make release-major` and in CHANGELOG.md only two lines are added. 
`+`
`+## [1.0.0] - 2017-02-23`

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased][unreleased]
+## [Unreleased]
```
Comment (jfilak) [inline CHANGELOG.md]: The lower case version makes a link. See the last line, you need to make the first letter capital there.

**Comment by jfilak** [comment]
> Ok, I updated it. Now it works better. make release-major and in CHANGELOG.md only two lines are added.

The script should update the links at the end of `CHANGELOG.md` too. So, at least 3 lines should be added and 1 changed. The 2 you have mentioned plus
```[Unreleased]: https://github.com/abrt/faf/compare/1.0.0...HEAD```
plus
```[1.0.0]: https://github.com/abrt/faf/compare/0.12.300...1.0.0```

**Comment by marusak** [comment]
retest this please

**Comment by sorki** [comment]
Looks good, thanks.

**Pull request merged**